src/main/kotlin/commands/CapeCommand.kt

```diff
@@ -0,0 +1,463 @@
+                                return@doesLater
+                            }
+
+                            if (!hexRegex.matches(colorPrimary) || !hexRegex.matches(colorBorder)) {
+                                message.error("You must enter both colors in 6-long hex format, eg `9b90ff` or `8778ff`.")
+                                return@doesLater
+                            }
+
+                            changeTimeOut(capeUUID)?.let {
+                                message.error(changeError(capeUUID, it))
+                                return@doesLater
+                            }
+
+                            val oldCapeColor = cape.color
+                            cape.color = CapeColor(colorPrimary.toLowerCase(), colorBorder.toLowerCase())
+
+                            val oldColors = oldCapeColor.toEmoji()
+                            val newColors = cape.color.toEmoji()
+
+                            message.channel.send {
+                                embed {
+                                    description = "Successfully changed the colors of Cape `${cape.capeUUID}`!"
+                                    field(
+                                        "Old Colors",
+                                        oldColors,
+                                        true
+                                    )
+                                    field(
+                                        "New Colors",
+                                        newColors,
+                                        true
+                                    )
+                                    color = Colors.success
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+        }
+
+        literal("save") {
+            doesLaterIfHas(PermissionTypes.AUTHORIZE_CAPES) {
+                save()
+                commit()
+                message.success("Saved!")
+            }
+        }
+
+        literal("load") {
+            doesLaterIfHas(PermissionTypes.AUTHORIZE_CAPES) {
+                load()
+                message.success("Loaded!")
+            }
+        }
+
+        load()
+    }
+
+
+    private fun load() {
+        if (!File(capesFile).exists()) return
+        try {
+            Files.newBufferedReader(Paths.get(capesFile)).use { bufferedReader ->
+                val cacheList = Gson().fromJson<List<CapeUser>>(bufferedReader, object : TypeToken<List<CapeUser>>() {}.type)
+                capeUserMap.clear()
+                capeUserMap.putAll(cacheList.associateBy { it.id })
+            }
+        } catch (e: Exception) {
+            log("Error loading capes!")
+            e.printStackTrace()
+        }
+    }
+
+    fun save() {
+        val capeUsers = capeUserMap.values.toList()
+        if (!File(capesFile).exists()) File(capesFile).createNewFile()
+
+        Files.newBufferedWriter(Paths.get(capesFile)).use {
```

Comment:
`File(capesFile)` and `Paths.get(capesFile)` can be turned into one variable instead to avoid repeating instance creation.
```suggestion
        val file = File(capesFile)
        if (!file.exists()) File(capesFile).createNewFile()

        Files.newBufferedWriter(file).use {
```